content/hi/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,183 @@
+---
+title: कचरा संग्रहण
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="गार्बेज-कलेक्शन" length="short">}} यह
```

Comment:
@akash-kumar-saw For `glossary_definition` to work as expected we should localize the glossary entry for "garbage-collection" (i.e. [en/docs/reference/glossary/garbage-collection.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/garbage-collection.md)) and incorporate it into this pull request by adding to list of Hindi glossary items inside [/hi/docs/reference/glossary](https://github.com/kubernetes/website/tree/main/content/hi/docs/reference/glossary). Also, please ensure that `term_id` is in English.

```suggestion
{{<glossary_definition term_id="garbage-collection" length="short">}} यह
```